dev: yarn install fails in Windows and WSL

**Describe the bug**

```sh
yarn

error /.../Bazecor/node_modules/drivelist: Command failed.
Exit code: 127
Command: prebuild-install --runtime napi || node-gyp rebuild
Arguments: 
Directory: /.../Bazecor/node_modules/drivelist
```

This is due to my `wsl2` is missing `node-gyp`. It shows if I run `yarn --verbose`:

```sh
yarn --verbose

...
Command: prebuild-install --runtime napi || node-gyp rebuild
Arguments:
Directory: ...\Bazecor\node_modules\drivelist
Output:
prebuild-install warn install No prebuilt binaries found (target=8 runtime=napi arch=x64 libc= platform=win32)
'node-gyp' is not recognized as an internal or external command,
operable program or batch file.
```

The way to fix this is installing `node-gyp`: `npm i -g node-gyp`.

I can create a PR to add this in the readme.

in wsl:

```sh
../libusb/libusb/os/linux_udev.c:28:10: fatal error: libudev.h: No such file or directory
  28 | #include <libudev.h>
  |          ^~~~~~~~~~~
  compilation terminated.

# needs
apt-get install libudev-dev
```

--- a/README.md
+++ b/README.md
@@ -1,5 +1,4 @@
-Bazecor
-=======
+# Bazecor
 
 ![Bazecor screenshot](data/screenshot.png)
 
@@ -49,6 +48,25 @@ To do a production build, use `yarn run make`, or limit it to a particular OS:
 | Linux   |              | `yarn run make-lin`       |
 
 See the `scripts` section of `package.json` for more scripts.
+
+### Development on Windows
+
+If you are using Windows, you will need `node-gyp` and `python3`:
+
+```sh
+# install python3 using winget
+# note that python 3.12 doesn't work
+winget install python.python.3.10.0
+
+# install node-gyp
+npm install -g node-gyp
+```
+
+If you are using WSL (with ubuntu), you will also need `libudev-dev`:
+
+```sm
+sudo apt-get install libudev-dev
+```
 
 ## Additional Information
 

